### PR TITLE
Add what's new pages for ASP.NET Core docs

### PR DESCRIPTION
**Pull request by scottaddie**
Add "what's new" pages that mirror those for .NET at https://docs.microsoft.com/dotnet/whats-new/. Note that these pages are generated by some code Bill and I wrote.

[Internal review page](https://review.docs.microsoft.com/aspnet/core/whats-new/?view=aspnetcore-1.0&branch=pr-en-us-16595)

### CODE REVIEW

**Comment by scottaddie** [comment]
@BillWagner @Rick-Anderson @MaximRouiller Do we want README files to appear in the "what's new" pages? They're responsible for the build warnings on this PR.

File: aspnetcore/whats-new/2019-09.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+---
+title: "ASP.NET Core docs: What's new (September 2019)"
+description: "What's new in ASP.NET Core docs in September 2019."
+ms.date: 01/17/2020
+---
+
+# ASP.NET Core docs: What's new (September 2019)
```
Comment (Rick-Anderson) [inline aspnetcore/whats-new/2019-09.md]:
```suggestion
# ASP.NET Core docs: What's new September 2019
```

Comment (Rick-Anderson) [reply aspnetcore/whats-new/2019-09.md]: Per MT style guide, avoid () when you can.

File: aspnetcore/whats-new/2019-10.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
+---
+title: "ASP.NET Core docs: What's new (October 2019)"
+description: "What's new in ASP.NET Core docs in October 2019."
+ms.date: 01/17/2020
+---
+
+# ASP.NET Core docs: What's new (October 2019)
```
Comment (Rick-Anderson) [inline aspnetcore/whats-new/2019-10.md]:
```suggestion
# ASP.NET Core docs: What's new October 2019
```

**Comment by scottaddie** [comment]
@Rick-Anderson The more descriptive our PR titles moving forward, the better the descriptions in the generated pages.

**Comment by Rick-Anderson** [comment]
> @Rick-Anderson The more descriptive our PR titles moving forward, the better the descriptions in the generated pages.

Yes, we should do that. PR titles can be edited at any time, preferably before merging.

We can't have build warnings. Why don't you use an absolute link to the GitHub repo?

**Comment by scottaddie** [comment]
@Rick-Anderson Agreed. Once we've determined how to handle README file links in these pages, I can fix the build warnings.

**Comment by Rick-Anderson** [comment]
> I can either remove the README file links from the pages, or I can configure DocFx to handle them.

If they're useful, config DocFX. Are they localized?

File: aspnetcore/whats-new/2019-09.md (hunk 1 above)
Comment (MaximRouiller) [reply aspnetcore/whats-new/2019-09.md]: > # ASP.NET Core docs: What's new for September 2019

Adding `for` sounds better. No?

**Comment by MaximRouiller** [comment]
Looking into this now.

**Comment by MaximRouiller** [comment]
Alright, @scottaddie. All those `README.md` files you are adding seem to all be for Samples. They aren't technically docs. You may want to exclude any URLs that include `/samples/` in the path for this specific repository. 

If they can't be represented as a docs.microsoft.com page, they should not be on that list. If that list looks too dry, you can also play with the number `75` that @BillWagner set up initially. It doesn't have to be static but it should output content that changed significantly without giving too many false positives.

**Comment by scottaddie** [comment]
@MaximRouiller Thank you for your input. That's the direction I was leaning after giving this some more thought. I'll remove the README files from this PR and then adjust the app accordingly.

**Comment by MaximRouiller** [comment]
@scottaddie do you have a wished schedule for this PR to be merged?

**Comment by scottaddie** [comment]
@MaximRouiller I hope to merge it today. There are a few manual tweaks I'll be making to the generated Markdown files.

File: aspnetcore/whats-new/2019-09.md (hunk 1 above)
Comment (scottaddie) [reply aspnetcore/whats-new/2019-09.md]: Updating the code right now to incorporate this suggestion.

**Pull request merged**